### PR TITLE
embedded chainspecs

### PR DESCRIPTION
**Pull request by fixxxedpoint**
- both mainnet and testnet chainspecs in raw format embedded into aleph-node binary
- mainnet uses conditional compilation - feature mainnet
- backward compability - calling with `chain=<filepath>` loads a chainspec from a file
- default value = testnet (mainnet when compiled with feature mainnet)

### CODE REVIEW

File: bin/node/src/cli.rs [hunk 1]
```diff
@@ -44,11 +44,17 @@ impl SubstrateCli for Cli {
     }
 
     fn load_spec(&self, id: &str) -> Result<Box<dyn sc_service::ChainSpec>, String> {
+        #[cfg(not(feature = "mainnet"))]
+        let default_chain = "testnet";
+        #[cfg(feature = "mainnet")]
+        let default_chain = "mainnet";
```
Comment (artemis-cc) [inline bin/node/src/cli.rs:50]:
```suggestion
        let default_chain = if cfg!(feature = "mainnet") { "mainnet" } else { "testnet" };
```

**Comment by fixxxedpoint** [comment]
> I'm wondering if it would be better for the crate feature to turn on _support_ for connecting to the mainnet, but leave the _default_ set to the _testnet_ (still requiring `--chain=mainnet` on the command line to use the mainnet). Maybe I'm worrying too much, but I'm just imagining someone building with `--feature=mainnet` or even just `--all-features` without thinking, and suddenly they have a node on the mainnet by accident…

I think you're right and this conditional compilation thing is just useless here. I will change it to default = testnet.

File: bin/node/src/cli.rs (hunk 1 above)
Comment (fixxxedpoint) [reply bin/node/src/cli.rs]: Removed and using `testnet` as default value.

**Review by mike1729 — APPROVED** [review]

**Review by artemis-cc — APPROVED** [review]
:+1: Fairly simple and looks good to me.

**Pull request merged**